vector/src/main/java/im/vector/app/features/spaces/create/CreateSpaceViewModel.kt

```diff
@@ -79,28 +90,42 @@ class CreateSpaceViewModel @AssistedInject constructor(
                 }
                 _viewEvents.post(CreateSpaceEvents.NavigateToDetails)
             }
-            is CreateSpaceAction.NameChanged -> {
+            is CreateSpaceAction.NameChanged            -> {
+                val tentativeAlias =
+                        getAliasFromName(action.name)
+
                 setState {
                     copy(
                             nameInlineError = null,
-                            name = action.name
+                            name = action.name,
+                            aliasLocalPart = tentativeAlias,
```

Comment:
What if the user already have entered/edited the alias? I think it will be overridden by the tentative, which is not ideal. Check `aliasManuallyModified`?

Reply:
Will check how to make it cleaner, but currently formEditText only use this value once (it's the initial value) that's why it wasn't causing issues

Reply:
ok